### ISSUE TITLE
`workspace $ws output $out` doesn't work 

### ISSUE THREAD

**Issue by djmitche** [SirCmpwn/sway#1122]
```
 $ swaymsg -t get_outputs
[
   {
     "id": 1,
     "name": "eDP-1",
     "rect": {
       "x": 0,
       "y": 0,
       "width": 2560,
       "height": 1440
     },
     "visible": true,
     "focused": false,
     "active": true,
     "primary": false,
     "layout": "output",
     "type": "output",
     "current_workspace": "F1",
     "scale": 1
   },
   {
     "id": 3,
     "name": "HDMI-A-2",
     "rect": {
       "x": 2560,
       "y": 0,
       "width": 1360,
       "height": 768
     },
     "visible": true,
     "focused": false,
     "active": true,
     "primary": false,
     "layout": "output",
     "type": "output",
     "current_workspace": "F2",
     "scale": 1
   }]
 $ swaymsg -t get_workspaces
[
   {
     "id": 9,
     "name": "F1",
     "rect": {
       "x": 0,
       "y": 0,
       "width": 2560,
       "height": 1424
     },
     "visible": true,
     "num": -1,
     "output": "eDP-1",
     "urgent": false,
     "type": "workspace",
     "layout": "splith",
     "focused": false
   },
   {
     "id": 26,
     "name": "F8",
     "rect": {
       "x": 0,
       "y": 0,
       "width": 1360,
       "height": 752
     },
     "visible": false,
     "num": -1,
     "output": "HDMI-A-2",
     "urgent": false,
     "type": "workspace",
     "layout": "splith",
     "focused": false
   },
   {
     "id": 49,
     "name": "F2",
     "rect": {
       "x": 0,
       "y": 0,
       "width": 1360,
       "height": 752
     },
     "visible": true,
     "num": -1,
     "output": "HDMI-A-2",
     "urgent": false,
     "type": "workspace",
     "layout": "splith",
     "focused": true
   }
 ]
```

```
$ swaymsg workspace F2 output eDP-1
[
   {
     "success": true
   }
 ]
```

Expected behavior: F2 appears on eDP-1

Actual behavior: no change
```
 $ swaymsg -t get_workspaces
[
   ...
   {
     "id": 49,
     "name": "F2",
     "rect": {
       "x": 0,
       "y": 0,
       "width": 1360,
       "height": 752
     },
     "visible": true,
     "num": -1,
     "output": "HDMI-A-2",
     "urgent": false,
     "type": "workspace",
     "layout": "splith",
     "focused": true
   }
 ]
```

**Comment by djmitche** [comment]
Ah, and the source shows why:
```
	// TODO: Move workspace to output. (don't do so when reloading)
```

**Comment by zandrmartin** [comment]
Are you trying to move a workspace that already exists to a given output?

**Comment by zandrmartin** [comment]
If you are, the command is `move workspace to output <name>` - `workspace <name> output <output-name>` only assigns a workspace to a given output, it does not actually move it.

**Comment by djmitche** [comment]
No, the first mention of the workspace in the config is
```
    workspace F7 output $screen3
```
(where $screen2 is HDMI-A-2)

**Comment by SirCmpwn** [comment]
>where $screen2 is HDMI-A-2

But the config line you pasted uses $screen3

**Comment by djmitche** [comment]
fixed :)

**Comment by djmitche** [comment]
I didn't mean the issue is fixed, I meant hte comment was fixed.  The issue is still very much not fixed.

**Comment by SirCmpwn** [comment]
Wait so are you or are you not trying to move the workspace at runtime

**Comment by djmitche** [comment]
I'm not :)

I want F7 to always appear on HDMI-A_2.  The line above is in my config.

**Comment by SirCmpwn** [comment]
Can you provide a debug log?

**Comment by zandrmartin** [comment]
What version of Sway are you running? This actually _was_ broken in 0.12, but it was fixed in 0.12.1 - see #1105.

**Comment by djmitche** [comment]
Ah, indeed, 0.12.

2017-03-26 20:39 GMT-04:00 Zandr Martin <notifications@github.com>:
